docs/python-feature-support.md

```diff
@@ -27,53 +27,29 @@ menu:
 
 # Feature Support
 
-The goal is that the python library will provide a functional, performant subset of the java library. The initial focus has been on reading table metadata as well as providing the capability to both plan and execute a scan.
+The goal is that the python library will provide a functional, performant subset of the Java library. The initial focus has been on reading table metadata and provide a convenient CLI to go through the catalog.
 
 ## Feature Comparison
 
 ### Metadata
 
 | Operation               | Java  | Python |
 |:------------------------|:-----:|:------:|
-| Get Schema              |    X  |    X   |
-| Get Snapshots           |    X  |    X   |
-| Plan Scan               |    X  |    X   |
-| Plan Scan for Snapshot  |    X  |    X   |
+| Get Schema              |    X  |   X    |
+| Get Snapshots           |    X  |   X    |
+| Plan Scan               |    X  |        |
+| Plan Scan for Snapshot  |    X  |        |
 | Update Current Snapshot |    X  |        |
 | Set Table Properties    |    X  |        |
```

Comment:
Is this now supported, at least in the CLI?

Reply:
Missed that one, both in the CLI and Python 👍🏻